projects/laji/src/app/+observation/download/observation-download.component.ts

```diff
@@ -185,13 +185,7 @@ export class ObservationDownloadComponent implements OnDestroy {
 
   updateCount() {
     this.observationDataService.getData(this._originalQuery).pipe(
-      map(data => data.private.total),
-      catchError(() => this.warehouseService.warehouseQueryCountGet({
-        ...this.query,
-        secured: true
-      }).pipe(
-        map(result => result.total)
-      ))
+      map(data => data.securedCount)
     ).subscribe(total => {
       this.privateCount = total;
```

Comment:
`privateCount` is just a copy of `secureCount`. `privateCount` could be removed?

Reply:
Yeah it'd require some refactoring to get this working with observables instead of the class instance variables... It's ok as it is :+1: